### DISCUSSION TITLE
Can Multiple Users Share Custom Domain?

### DISCUSSION THREAD

**Discussion by hcurtiss** [simple-login/app#1359] [Q&A]
All four family members signed up (expensive!). Can we all share the same custom domain?

**Comment by nguyenkims** [comment]
@hcurtiss currently a custom domain can only be used by a single account. However there're workarounds for this:
- each family member can have their own subdomain
- you can also create an alias using the domain and *transfer* it to another user.

**Reply by hcurtiss** [reply]
@nguyenkims I purchased four separate SL accounts, one for each of us.  Is it possible for me to set each of them up with a subdomain using my custom domain?  That is, I will use family.io as my custom domain.  And I'd like each of them to use abc.family.io, def.family.io, and ghi.family,io.  Is that possible on four separate accounts?  I don't really want the teenagers using my SL login to monitor/control their aliases.

**Reply by hcurtiss** [reply]
So, I confirmed that this works.  For the other accounts, I just setup a new custom domain using the subdomain.  The [subdomain instructions for namecheap](https://simplelogin.io/docs/custom-domain/registrars/namecheap/namecheap-subdomain/) worked well.  I just used the first letter of the first name of each family member as the sub.  This is exactly what I needed!

**Reply by nguyenkims** [reply]
@hcurtiss Cool, glad that the workaround works!

**Comment by Freeedim** [comment]
I support @hcurtiss here.

Although I acknowledge the workaround works, it is not ideal for adoption, especially by non-tech people, when you extend the domain part of their email address. The extension is either short and meaningless, therefore awkward to many, or long enough to be a hassle.

Would it be hard to just allow unrestricted domains, and checking a different sl-verification value for each user (SL account) to verify they have admin access to the DNS? I am quite a beginner in DNS / email stuff, so please excuse me if it's irrelevant.
